keyboards/yetis/yetis.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2023 Norbert Pocs (norbertpocs0@gmail.com, jogme)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "quantum.h"
+
+#ifdef OLED_ENABLE
+/* corresponds to the layer number in keymaps/default/keymap.c */
+#define LAYER_GAME 4
+oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
+    if (!is_keyboard_master()) {
+        return OLED_ROTATION_90;
+    }
+    return OLED_ROTATION_270;
+}
+
+static void render_logo(void) {
```

Comment:
It might be worth not making these functions static so that the end users can re-use these functions.  But this is more of a style/behavior nitpick.

Reply:
fixed